os/posix/NetworkSocketPosix.cpp

```diff
@@ -64,6 +64,7 @@ void NetworkSocketPosix::SetMaxPriority(){
 	if(res<0){
 		LOGE("error setting darwin-specific net priority: %d / %s", errno, strerror(errno));
 	}
+#elif defined(__FreeBSD__)
```

Comment:
I don't quite understand what do you want by this, LOGI seem to be of info level and shouldn't be fatal

Reply:
Make code more explicit and self documenting - such sockopt is OS specific and doesn't exist in FreeBSD. Also reduces console spam. If you insist - can be removed.

Reply:
Yeah, I don't think that's really needed. If console spam is a problem for you then adjust default log level in a downsteam patch.